blog/exporters.md

```diff
@@ -176,18 +234,18 @@ In the `secadvisor.yml` file, uncomment the `iam_endpoint` field and set it to `
 
 For running tests on a local machine, it is possible to set up a local Minio object store.
 
-For details, see the instructions found at [Secadvisor Developer's README](https://github.com/skydive-project/skydive/tree/master/contrib/pipelines/secadvisor/README.md).
+For details, see the instructions found at [Secadvisor Developer's README](https://github.com/skydive-project/skydive/tree/master/contrib/exporters/secadvisor/README.md).
 
 ## Deploy the Security Advisor pipeline
 
-Build and run the pipeline in the secadvisor directory: $SKYDIVE_BASE/contrib/pipelines/secadvisor.
+Build and run the pipeline in the secadvisor directory: $SKYDIVE_BASE/contrib/exporters/secadvisor.
```

Comment:
s/pipeline/exporter/

Reply:
I thought pipeline directory is being removed from the path. The current path of the source is $SKYDIVE_BASE/contrib/exporters/secadvisor.

Reply:
I meant fix:

```
Build and run the pipeline in the secadvisor directory: 
```

To:

```
Build and run the exporter in the secadvisor directory: 
```

Reply:
OK